### PR TITLE
Release 22.11.0-1

### PR DESCRIPTION
**Pull request by jezdez**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Just trying to figure out if https://github.com/conda-forge/tqdm-feedstock/pull/114 might be the culprit what made https://github.com/conda-forge/admin-requests/pull/523 needed.

### CODE REVIEW

**Comment by jezdez** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -58,6 +58,7 @@ requirements:
     - setuptools >=31.0.1
     - toolz >=0.8.1
     - pluggy >=1.0.0
+    - importlib_resources
```
Comment (jakirkham) [inline recipe/meta.yaml:61]: Does this need a selector? Does it get skipped for certain Python versions that have it builtin? Or is it always needed?

Comment (jezdez) [reply recipe/meta.yaml:61]: it's not needed by conda at all, this is related to tqdm being added as a dependency and https://github.com/conda-forge/tqdm-feedstock/pull/114 being open

**Comment by jezdez** [comment]
Just removed importlib_resources again to see if it goes back to failure state on the advise of @beckermr

**Comment by beckermr** [comment]
Yup. That was something transient.

**Comment by beckermr** [comment]
Well whatever it is, it's gone now.

**Comment by jezdez** [comment]
Dunno what to do now, all seems alright, and yet

**Review by mbargull — CHANGES_REQUESTED** [review]
The now passing build was expected.
The previously failing builds used `conda[channel=conda-forge,subdir=win-64,version=22.11.0,build=py39hcbf5309_0]` in `base`.
We mark all `22.11.0` builds, including the aforementioned one, as `broken`.
As such, all current builds will use `conda=22.9.0` again and not reproduce the error.
To reproduce, you have to install `conda-forge/label/broken::conda=22.11.0` into `base`.

**Comment by mbargull** [comment]
The solver issue should be fixed with https://github.com/conda/conda/pull/12148

**Comment by beckermr** [comment]
Great work! It's always something funny.

**Comment by mbargull** [comment]
superseded by gh-192

**Pull request closed without merging**